test/Feature/BFSSearcherAndDFSSearcherInterleaved.c

```diff
@@ -0,0 +1,42 @@
+// RUN: %llvmgcc %s -emit-llvm -g -c -o %t.bc
+// RUN: rm -rf %t-bfs.klee-out
+// RUN: rm -rf %t-dfs.klee-out
+// RUN: rm -rf %t-bfs-dfs.klee-out
+// RUN: rm -rf %t-dfs-bfs.klee-out
+// RUN: %klee -libc=uclibc -output-dir=%t-bfs.klee-out -search=bfs %t.bc >%t-bfs.out
+// RUN: %klee -libc=uclibc -output-dir=%t-dfs.klee-out -search=dfs %t.bc >%t-dfs.out
+// RUN: %klee -libc=uclibc -output-dir=%t-bfs-dfs.klee-out -search=bfs -search=dfs -use-batching-search %t.bc >%t-bfs-dfs.out
+// RUN: %klee -libc=uclibc -output-dir=%t-dfs-bfs.klee-out -search=dfs -search=bfs -use-batching-search %t.bc >%t-dfs-bfs.out
+// RUN: cat %t-bfs.out | grep -q "ABCD"
```

Comment:
Could you replace these checks (using cat and grep) with the `FileCheck` command? You'd have something like

```c++
RUN: FileCheck -input-file=%t-bfs.out -check-prefix=BFS-CHECK %s
RUN: FileCheck -input-file=%t-dfs.out -check-prefix=DFS-CHECK %s
RUN: FileCheck -input-file=%t-bfs-dfs.out -check-prefix=BFS-DFS-CHECK %s
RUN: FileCheck -input-file=%t-dfs-bfs.out -check-prefix=DFS-BFS-CHECK %s
```


and then a set of source lines (where ever you think most appropriate)

```c++
// BFS-CHECK: ABCD
// DFS-CHECK: DCBA
// BFS-DFS-CHECK: ABCD
// DFS-BFS-CHECK: ACDB
```